### PR TITLE
[main][deploy] preparing script for deployment xalpaca on ftm

### PR DESCRIPTION
**Pull request by blackalpacaa**
## Description

- Create deployment script of AlpacaFeeder02.
- Adjust command for deployment script.

## Why?

- Preparing script for deployment xalpaca on ftm

### CODE REVIEW

File: deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    alpacaGrassHouseAddress.address,
+  ])) as AlpacaFeeder02;
+  await alpacaFeeder.deployed();
+  nonce++;
+  console.log(`>> Deployed alpacafeeder02 at ${alpacaFeeder.address}`);
+  console.log("✅ Done");
+
+  console.log(">> Transferring ownership and set okHolders of proxyToken to be alpacaFeeder");
+  const proxyToken = ProxyToken__factory.connect(config.Tokens.fdALPACA, deployer);
+  await proxyToken.setOkHolders([alpacaFeeder.address, config.MiniFL.address], true, { nonce: nonce++ });
```
Comment (sora-alpaca) [inline deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts:52]: if nonce on line 29 is 10
what is this nonce should be?

File: deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  await proxyToken.transferOwnership(alpacaFeeder.address, { nonce: nonce++ });
```
Comment (sora-alpaca) [inline deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts:53]: if nonce on line 29 is 10
what is this nonce should be?

File: deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  console.log("✅ Done");
+
+  console.log(">> Sleep for 10000msec waiting for alpacaFeeder to completely deployed");
+  await new Promise((resolve) => setTimeout(resolve, 10000));
+  console.log("✅ Done");
+
+  console.log(">> Depositing proxyToken to MiniFL pool");
+  await alpacaFeeder.miniFLDeposit({ nonce });
```
Comment (sora-alpaca) [inline deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts:61]: if nonce on line 29 is 10
what is this nonce should be?

File: deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  const config = ConfigEntity.getConfig();
+  const deployer = (await ethers.getSigners())[0];
+  let nonce = await deployer.getTransactionCount();
+
+  const alpacaGrassHouseAddress = config.GrassHouses.find((gh) => gh.name === "ALPACA");
+  if (alpacaGrassHouseAddress === undefined) throw new Error(`could not find ALPACA GrassHouse`);
+  if (config.Tokens.fdALPACA === undefined) throw new Error(`could not find config.Tokens.fdALPACA`);
+  if (config.MiniFL === undefined) throw new Error(`could not find config.MiniFL`);
+
+  console.log(`>> Deploying AlpacaFeeder`);
```
Comment (jr-alpaca) [inline deploy/exec/alpaca-feeder/deploy/alpaca-feeder02.ts:36]:
```suggestion
  console.log(`>> Deploying AlpacaFeeder02`);
```

**Review by jr-alpaca — APPROVED** [review]

**Review by alpanaca — APPROVED** [review]

**Pull request merged**